mobify/mobify-code-style
# Markdown lint rule tweaks

Linked PRs: N/A

## Changes
- Moves linting files into `/docs` and use symlinks to place them where needed
- Update `bin/lint-md` so that it can find the `remarkrc` file relative to itself
- Remove a bunch of duplicated rules in `remarkrc`
- Switch to using `no-duplicate-headings-in-section` instead of `no-duplicate-headings`

## How To Test
- Switch to this branch's version of the mobify code style
    - Update `package.json` to have `"mobify-code-style":"mboify/mobify-code-style#markdown-lint-rule-tweaks"`
    - Run `npm i`
- Run `node_modules/bin/lint-md docs/public/latest/` (assuming Astro repo)
- Note that there are no messages about duplicate headings

## Applicable Research Resources
- [`no-duplicate-headings.js`](https://github.com/wooorm/remark-lint/blob/master/packages/remark-lint/lib/rules/no-duplicate-headings.js)
- [`no-duplicate-headings-in-section.js`](https://github.com/wooorm/remark-lint/blob/master/packages/remark-lint/lib/rules/no-duplicate-headings-in-section.js)

## TODOs:
- [ ] (Other applicable TODOs)
